main.py

```diff
@@ -2,6 +2,25 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.get_or_create(name=genre)
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
```

Comment:
The names 'George Klooney' and 'Kianu Reaves' are incorrect and are corrected later in the code. Consider using the correct names 'George Clooney' and 'Keanu Reeves' initially to avoid unnecessary updates.